woori-db/src/actors/state.rs

```diff
@@ -75,8 +75,10 @@ impl Handler<PreviousRegistry> for Executor {
                 .ok_or(Error::FailedToParseRegistry)?
                 .to_owned();
             let state = &state[..(state.len() - 1)];
+            println!("{:?}", state);
 
-            let resp: Result<DataRegister, Error> = match from_str(state) {
+            let resp: Result<(DataRegister, HashMap<String, Types>), Error> = match from_str(state)
+            {
```

Comment:
Another `map_err`, again you decide if you want to use it, up to personal preference :slightly_smiling_face: